### DISCUSSION TITLE
Markdown Mermaid Diagrams Do Not Support FontAwesome Icons

### DISCUSSION THREAD

**Discussion by jamesward** [community/community#11940] [Other Feature Feedback, Questions, & Ideas]
In normal Mermaid you can add FontAwesome icons like:
```
graph TD
    foo(fa:fa-bug foo)
```
![image](https://user-images.githubusercontent.com/65043/155216448-4e97b9a1-50da-4f40-86d3-1ddf2b6000c5.png)

But that doesn't work with GitHub's Mermaid:
```mermaid
graph TD
    foo(fa:fa-bug foo)
```

It'd be great if FontAwesome Icon support was added.

**Comment by kthy** [comment]
+1

Currently GitHub strips the FontAwesome stylesheet `<link>` element from my markdown files when rendering them, which, fair cop, but would be nice to have FA support baked in.

**Reply by priyasingh199885** [reply]
is there a solution yet?

**Reply by vincerubinetti** [reply]
I guess I can understand why they wouldn't want to include it, since importing all the icons (naive approach) is like 1 MB of payload. But it'd be really nice to have more options than what [unicode emojis give us](https://emojipedia.org/). For example, there's no really good emoji's for code-related things like "</>", "database", "bug", "i/o", etc.

**Comment by taffarel55** [comment]
It would be great if it did, the integration has to be complete!

**Comment by t2hv33** [comment]
Please support this feature
